backend/prisma/seed.ts

```diff
@@ -16,8 +28,140 @@ async function main(): Promise<void> {
       data: user_security,
     });
   }
+
+  for (const bodyType of bodyTypes) {
+    const dataToSeed = new AutoriaPlainDataDto(bodyType);
+    await prisma.body_Type.create({
+      data: dataToSeed,
+    });
+  }
+  for (const manufacturer_country of manufacturers) {
+    const dataToSeed = new AutoriaPlainDataDto(manufacturer_country);
+    await prisma.manufacturer_Country.create({
+      data: dataToSeed,
+    });
+  }
+  for (const color of colors) {
+    const dataToSeed = new AutoriaPlainDataDto(color);
+    await prisma.color.create({
+      data: dataToSeed,
+    });
+  }
+  for (const drivetrain of drivetrains) {
+    const dataToSeed = new AutoriaPlainDataDto(drivetrain);
+    await prisma.drivetrain.create({
+      data: dataToSeed,
+    });
+  }
+  for (const fuelType of fuelTypes) {
+    const dataToSeed = new AutoriaPlainDataDto(fuelType);
+    await prisma.fuel_Type.create({
+      data: dataToSeed,
+    });
+  }
+  for (const transmissionType of transmissionTypes) {
+    const dataToSeed = new AutoriaPlainDataDto(transmissionType);
+    await prisma.transmission_Type.create({
+      data: dataToSeed,
+    });
+  }
+
+  for (const option of optionsTypes) {
+    await prisma.option.create({
+      data: {
+        'name': option.name,
+        'autoria_code': option.value,
+        'type': option.type as OptionType,
```

Comment:
Why would we use string properties in the JS object code?

```suggestion
        name: option.name,
        autoria_code: option.value,
        type: option.type as OptionType,
```